### PR TITLE
Konstantinos / Responsive issue Tradertools

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [x] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/elements/dropdown.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { Flex } from 'components/containers'
 
 const Symbol = styled(Flex)`
     width: fit-content;
-    margin-top: 8px;
+    margin-top: px;
```
Comment (prince-deriv) [inline src/components/elements/dropdown.js:12]: Is this supposed to be `0`?

File: src/components/elements/dropdown.js [hunk 2]
```diff
@@ -40,6 +40,9 @@ const Symbol = styled(Flex)`
 `
 
 export const DropdownContainer = styled.ul`
+    @media (width: 428px) {
+        height: 46px;
+    }
```
Comment (prince-deriv) [inline src/components/elements/dropdown.js:45]: can we use one of the sizes from `device.ts` ?

File: src/pages/trader-tools/pnl-for-multipliers/_pnl_multipliers_calculator.tsx [hunk 3]
```diff
@@ -1689,13 +1689,13 @@ const PnlMultipliersCalculator = () => {
                                             'Let’s say you want to calculate the take profit amount when you want to open a position for Volatility 100 Index priced at 3376.24 USD with a stake amount of 10 USD, a multiplier value of x100, and a take profit level of 3400 in Up direction.',
                                         )}
                                     </Text>
-                                    <Show.Desktop max_width="mobileL">
+                                    <Show.Desktop max_width="tablet">
```
Comment (prince-deriv) [inline src/pages/trader-tools/pnl-for-multipliers/_pnl_multipliers_calculator.tsx:1692]: Can we also refactor this with the new `show` component under `components/containers/visibility.tsx`

File: src/components/elements/dropdown.js (hunk 1 above)
Comment (konstantinosG-deriv) [reply src/components/elements/dropdown.js:12]: No, I forgot to put it back to its original value after checking some stuff. Thanks

File: src/components/elements/dropdown.js (hunk 2 above)
Comment (konstantinosG-deriv) [reply src/components/elements/dropdown.js:45]: Yes

File: src/pages/trader-tools/pnl-for-multipliers/_pnl_multipliers_calculator.tsx (hunk 3 above)
Comment (konstantinosG-deriv) [reply src/pages/trader-tools/pnl-for-multipliers/_pnl_multipliers_calculator.tsx:1692]: Can't find `Show` component in visibility.tsx, maybe you meant components/containers/show.js? Isn't it already using the show component with Show.Desktop?

Comment (vijayasree-deriv) [reply src/pages/trader-tools/pnl-for-multipliers/_pnl_multipliers_calculator.tsx:1692]: @konstantinosG-deriv , import Desktop from visibility.tsx and replace <Show.Desktop max_width="tablet"> with <Desktop breakpoint="tablet">.Similarly for Mobile as well.

Comment (konstantinosG-deriv) [reply src/pages/trader-tools/pnl-for-multipliers/_pnl_multipliers_calculator.tsx:1692]: @vijayasree-deriv  Oh okay 👍

File: src/pages/trader-tools/pnl-for-margin/_pnl-margin-calculator.tsx [hunk 4]
```diff
@@ -403,8 +403,8 @@ const PnlMarginCalculator = () => {
 
                                 return (
                                     <>
-                                        <Show.Desktop max_width="mobileL">
-                                            <CalculatorForm>
+                                        <Desktop breakpoint={'tablet'}>
```
Comment (vijayasree-deriv) [inline src/pages/trader-tools/pnl-for-margin/_pnl-margin-calculator.tsx:406]: @konstantinosG-deriv the breakpoint should be mobileL

File: src/pages/trader-tools/common/_style.ts [hunk 5]
```diff
@@ -87,19 +87,25 @@ const swap_tab_selector_css = (props) =>
           `
 
 export const SwapTabSelector = styled(Flex)`
-    padding: 22px 64px;
-    width: auto;
+    @media ${device.mobileL} {
+        width: 160px;
+    }
```
Comment (NikitK-deriv) [inline src/pages/trader-tools/common/_style.ts:92]: Seems media query rules should be after `${swap_tab_selector_css}`

File: src/pages/trader-tools/common/_style.ts [hunk 6]
```diff
@@ -149,11 +155,14 @@ export const CalculatorForm = styled(Form)`
 `
 export const PnlCalculatorFormMobile = styled(Form)`
     ${CalculatorFormStyles}
-    width: 328px;
+    width:43rem;
```
Comment (NikitK-deriv) [inline src/pages/trader-tools/common/_style.ts:158]: Let's make permutation here
1. css rules
2. ${CalculatorFormStyles}
3. media query

Comment (konstantinosG-deriv) [reply src/pages/trader-tools/common/_style.ts]: Ok

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**